draft-packages/badge/KaizenDraft/Badge/Badge.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import { cleanup, render, screen } from "@testing-library/react"
+import * as React from "react"
+
+import { Badge, BadgeProps } from "./Badge"
+
+afterEach(cleanup)
+
+const renderBadge = (props?: BadgeProps) => {
+  const mergedBadgeProps = { ...props }
```

Comment:
Is this a leftover from a previous approach? Nothing seems to get merged here? The `props` just get splatted into another object.